src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,13 @@
 
 Release History
 ===============
+0.4.64
++++++
+* Add support for Open Service Mesh as an AKS addon
+* Add support for proxy certificate in Connectedk8s
+* Add support to get available upgrade versions for an agent pool in AKS
+* Retire the ai-did-you-mean-this extension
```

Comment:
What is this?

Reply:
Based on commit `9de47aec [ai-did-you-mean-this] Make the extension retired (#2402)`, thought it would be worth noting in the release notes.

Reply:
is this need to be in ak-preview extension?

Reply:
hi @gtracer could you pls fix this, then I can release the extension

Reply:
This is just in the release notes. Since this change is a part of the same repo, if we end up cutting a release for the cli this commit would also be a part of it.

That being said, I am not familiar with the release process of azure cli extensions, so please feel free to update this PR as necessary to cut a release.

Reply:
@yungezz I have removed the unrelated release notes.